syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -67,6 +68,8 @@ def reconstruct(self):
         plain_text = self.__map_modular_to_real(plain_text_mod)
         return plain_text
 
+    get = reconstruct
```

Comment:
how is that useful?

Reply:
a.reconstruct() <-- this is a wrapper
and in native we do not have the reconstruct method
so it fails, thus as I mentioned earlier we should use .get() universally in future.

Reply:
Hmm...thinking of this. Could we add a reconstruct method in "native" and have a check -- we can do the reconstruct only if we have ```AdditiveSharingTensor``` or ```ReplicatedSharingTensor```

Reply:
fixed in latest commit!